scss/functions/_units.scss

```diff
@@ -6,5 +6,5 @@
 
 // Strip units from passed value
 @function stripUnit($n) {
-  @return $n / ($n*0+1);
+    @return $n / ($n*0+1);
```

Comment:
`$n*0+1` should be written with a single space on each side of the operator: `$n*0 + 1`<br>`$n*0` should be written with a single space on each side of the operator: `$n * 0`